Examples/LocationManager/LocationManager/LocalSearchClient/LocalSearchModels.swift

```diff
@@ -0,0 +1,214 @@
+import MapKit
+
+struct LocalSearchResponse: Equatable {
```

Comment:
This is a struct version of `MKLocalSearch.Response` in order to work around the fact that it is not `Equatable` _and_ is not constructible, and hence is basically impossible to test with 😟